Sources/Utility/Git.swift

```diff
@@ -12,52 +12,44 @@ import class Foundation.ProcessInfo
 import Basic
 
 extension Version {
-    static func vprefix(_ string: String) -> Version? {
-        if string.first == "v" {
-            return Version(string: String(string.dropFirst()))
-        } else {
-            return nil
+    init?(tag: String) {
+        if tag.first == "v" {
+            self.init(string: String(tag.dropFirst()))
+        }
```

Comment:
Nitpick: can you put else on the same line as the if closing brace to follow the project's informal style?